mne/viz/_3d.py

```diff
@@ -3304,13 +3304,21 @@ def set_3d_options(antialias=None):
         False is useful when renderers have problems (such as software
         MESA renderers). This option can also be controlled using an
         environment variable, e.g., ``MNE_3D_OPTION_ANTIALIAS=false``.
+    depth_peeling : bool | None
+        If not None, set the default setting for accurate transparency.
```

Comment:
Argh I **hate** the phrasing, "not None", I always have to do massive mental gymnastics to understand what it means (I think we discussed this the other day 🙈) Do you think you could turn this around?

Reply:
I reworked it in [3b06ba5](https://github.com/mne-tools/mne-python/pull/10319/commits/3b06ba5e1e5748555a284cecfc6cda2cef66b96e).